packages/dcos-log/extra/dcos-log-master.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=DC/OS Log service master: DC/OS logging service
+Description=DC/OS Log Master: exposes component, container, and task logs
```

Comment:
My understanding is that the master and agent component logs are reported to journald and exposed through dcos-log. And the task/container logs on the agents are also reported to journald and exposed through dcos-log. So while the agents just expose via internal API, the masters expose via external API. So the masters expose all 3 log types: component, container, and task.